src/commands/app/create.ts

```diff
@@ -73,6 +74,9 @@ export default class Create extends BaseCommand<typeof Create> {
       char: "d",
       description: commonMsg.CURRENT_WORKING_DIR,
     }),
+    "boilerplates": flags.string({
+      description: appCreate.BOILERPLATE_TEMPLATES,
+    }),
```

Comment:
@harshithad0703 , Since the user can't select more than one boilerplate at once, please alter the name to boilerplate.

Reply:
ok